pkg/Microsoft.Windows.Compatibility/tests/publishcompatibilityassets.ps1

```diff
@@ -0,0 +1,70 @@
+$repoRoot = ((get-item $PSScriptRoot).parent.parent.parent.FullName);
+$winRID = "win-x64";
+$dotnetPath = -join($repoRoot, "\Tools\dotnetcli\dotnet.exe")
+$csprojPath = -join($PSScriptRoot, "\", (Get-ChildItem $PSScriptRoot"\*.csproj" | Select-Object -ExpandProperty Name))
+$packagesCachePath = -join($repoRoot, "\packages")
+$localPackageSourcePath = -join($repoRoot, "\bin\packages\Debug\")
+$packageName = "Microsoft.Windows.Compatibility"
+
+function _pathExists($path)
+{
+	return Test-Path -Path $path
+}
+
+if (!(_pathExists $localPackageSourcePath))
+{
+	$localPackageSourcePath = -join($repoRoot, "\bin\packages\Release\")
+	if (!(_pathExists $localPackageSourcePath))
+	{
+		Write-Error -Message "Local package source must exist.";
+		Exit;
+	}
+}
+
+function _getPackageVersion()
+{
+	$searchPattern = -join($localPackageSourcePath, $packageName, ".[0-9].[0-9].[0-9]*.nupkg")
+	if (!(_pathExists $searchPattern))
+	{
+		Write-Error -Message (-join("Didn't find package: Microsoft.Windows.Compatibility in source: ", $localPackageSourcePath, " please run build -allConfigurations"))
+		Exit;
+	}
+
+	if (!((get-item $searchPattern).FullName -match '([0-9].[0-9].[0-9][-a-z0-9]*)'))
+	{
+		Write-Error -Message "Package name is invalid"
+		Exit;
+	}
+
+	return $matches[0]
+}
+
+function _restoreAndPublish($targetFramework, $outputType, $rid, $runtimeFramework, $refDirName)
+{
+	$packageVersion = _getPackageVersion
+    & $dotnetPath restore --packages $packagesCachePath /p:RestoreSources="https://api.nuget.org/v3/index.json;$localPackageSourcePath" /p:TargetFramework=$targetFramework /p:CompatibilityPackageVersion=$packageVersion $csprojPath
+    & $dotnetPath publish -r $rid /p:RestoreSources="https://api.nuget.org/v3/index.json;$localPackageSourcePath" /p:TargetFramework=$targetFramework /p:CompatibilityPackageVersion=$packageVersion /p:RuntimeFrameworkVersion=$runtimeFramework $csprojPath
+
+    $outputPath = -join($PSScriptRoot, "\bin\Debug\", $targetFramework, "\", $rid, "\publish\refs\")
+
+    if (!(_pathExists $outputPath))
+    {
+    	Write-Error -Message (-join("There was an error while publishing for framework: ", $targetFramework))
+		Exit;
+    }
+
+	Write-Output (-join("Published succedded for: ", $targetFramework))
+	
+	$refPath = -join($repoRoot, "\bin\ref\ApiPort\", $refDirName)
+
+	if (_pathExists $refPath)
+	{
+		Remove-Item $refPath -r -force
+	}
+
+	New-Item $refPath -ItemType directory
+	Copy-Item (-join($outputPath, "*")) $refPath
+}
+
+_restoreAndPublish "netcoreapp2.0" "Exe" $winRID "2.0.0" ".NET Core App 2.0 + Microsoft.Windows.Compatibility"
```

Comment:
I suspect all the spaces in the paths may cause some trouble I suggest something like `netcoreapp20_compat` and `netstandard20_compat`

Reply:
Ok. Didn't ApiCatalog show the supported framework as the folder name?